tracdap-services/tracdap-svc-meta/src/test/java/org/finos/tracdap/svc/meta/api/MetadataSearchApiTest.java

```diff
@@ -567,30 +623,6 @@ void emptySearchResult() {
         assertEquals(0, searchResult.getSearchResultCount());
     }
 
-    @Test
-    void invalidSearch_nullParams() {
-
-        var searchRequest = MetadataSearchRequest.newBuilder()
```

Comment:
I think we want to keep the first half of this test? Just get rid of the second part?